### ISSUE TITLE
Use IMDB weighted average rating instead of raw average.

### ISSUE THREAD

**Issue by kamou** [popcorn-official/popcorn-desktop#1659]
seems like as of today, popcorntime is not using IMDB's weighted average rating. but instead some raw average, leading to wrong rating for lots of movies.

take as an example the movie "daddy's girl":
popcorn time rates it 7.8 whereas IMDB actually rates it 4.7 (https://www.imdb.com/title/tt1662546/ratings)

**Comment by kiriles90** [comment]
Those are not iMDB ratings, they are Trakt.tv ratings.  

It might be a bit confusing but there is a subtle dot/spacer between the 'Open iMDB' page and the rating (it is like this for a long time, 2016 as far as I remember and maybe even before that)

**Comment by kamou** [comment]
ok ! Then let me suggest an improvement on the interface: what about providing a rating button just like the "show cast" button, but for ratings, providing ratings from different sources like imdb, trackt.tv, rotten tomatoes, etc... Because it seems that trackt.tv ratings are not always reliable. that would avoid having to click the imdb button each time. I'm sure I'm not the only one who does that.

**Comment by kiriles90** [comment]
@kamou Well that if it can be done in a way that the multiple ratings wont be confusing might not be a bad idea. Maybe someone can pitch in tell us what they think? Btw if you want change the title/description to this and @Persei08 or @team-pct can tag it as a feature request?

**Comment by tuesd4y** [comment]
Maybe adding a trakt.tv icon or some text as in this quick mockup here before the trakt.tv rating could make it more clear where the ratings are coming from. 

Also, adding the imdb rating after it's icon doesn't look to confusing to me, but I think that would mean one API call to imdb whenever that screen is opened.

Any thoughts on the UI or API part, @kamou @kiriles90 ?

![Screenshot 2020-09-06 at 14 44 46](https://user-images.githubusercontent.com/13107179/92326043-8bfe4180-f04f-11ea-8187-34c2aaff20e2.png)

**Comment by Martibis** [comment]
As a sidenote, it would be nice to actually get the rating from IMDB if they expose those easily.

**Comment by medmek** [comment]
Yeah good idea ! And maybe if it could be done in the same feature : 
Allow the user to store the setting of the ordered list of rating providers he would like to see in the details, and the top one in this list would be the user's favorite and thus used for the sorting in the catalog (or search)
